lib/Conversion/TritonGPUToLLVM/MemoryOpToLLVM.cpp

```diff
@@ -121,33 +121,12 @@ struct LocalLoadOpConversion : public ConvertOpToLLVMPattern<LocalLoadOp> {
 
   // FIXME [Dot LL]
   // Do for all DotOperandEncodingAttr once we have LLs for all of them
```

Comment:
The only thing missing is probably `DotOperandEncoding<BlockedEncodingAttr>`

Reply:
@binarman is working on this one. After that, I'll start moving DistributedLayout to inherit from LL